go/ql/lib/semmle/go/dataflow/internal/DataFlowImpl1.qll

```diff
@@ -1,5 +1,5 @@
 /**
- * DEPRECATED: Use `Global` and `GlobalWithState` instead.
+ *  DEPRECATED: Use `Global` and `GlobalWithState` instead.
```

Comment:
Why add an extra space here?

Reply:
I suppose this last commit was just ther to trigger CI and will be removed in the end..

Reply:
Ah, I didn't look at the commits 😆 . That makes sense.

Reply:
Indeed :-)